tests/emitters/emitter_base_test.py

```diff
@@ -71,6 +72,251 @@ def test_ask_emits_correct_num_sols_on_nonempty_archive(emitter_fixture):
     assert solutions.shape == (batch_size, len(x0))
 
 
+#
+# tell()
+#
+
+
+@pytest.fixture(
+    params=["GradientArborescenceEmitter", "EvolutionStrategyEmitter"])
+def shape_test_fixture(request):
+    """Creates an archive, emitter, and initial solution.
+
+    Returns:
+        Tuple of (archive, emitter, batch_size, solution_dim).
+    """
+    emitter_name = request.param
+
+    batch_size = 3
+    archive = GridArchive(solution_dim=1, dims=[10], ranges=[(-1.0, 1.0)])
+    if emitter_name == "GradientArborescenceEmitter":
+        emitter = GradientArborescenceEmitter(archive,
+                                              x0=np.array([0]),
+                                              sigma0=1.0,
+                                              step_size=0.1,
+                                              batch_size=batch_size)
+    elif emitter_name == "EvolutionStrategyEmitter":
+        emitter = EvolutionStrategyEmitter(archive,
+                                           x0=np.array([0]),
+                                           sigma0=1.0,
+                                           ranker="imp",
+                                           batch_size=batch_size)
+
+    return archive, emitter, batch_size
+
+
+@pytest.mark.parametrize("offset", [(0, 1), (1, 0), (1, 1)])
+def test_tell_wrong_shape_solution_batch(shape_test_fixture, offset):
```

Comment:
Parameterize with a parameter `wrong_array` that indicates which array to give a wrong shape to.